infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -110,6 +114,45 @@ public LogReplicationMetadataManager(CorfuRuntime rt, long topologyConfigId, Str
         setupTopologyConfigId(topologyConfigId);
     }
 
+    public void initializeReplicationStatusTable(String remoteClusterId) {
+        try {
+            IRetry.build(IntervalRetry.class, () -> {
+                try (TxnContext txn = corfuStore.txn(NAMESPACE)) {
+                    lock.lock();
```

Comment:
We use it currently in other methods that update the replication status to avoid race conditions with the polling task but that should not be an issue at the stage where this is called so we can remove it.